backend/sequelize/models/service.model.js

```diff
@@ -0,0 +1,30 @@
+const { DataTypes } = require("sequelize");
+
+module.exports = (db) => {
+  db.define("service", {
+    name: {

```

Comment:
Does sequelize creates an id field automatically?  Or do we need to create a primary key field to identify services uniquely?

Reply:
Yes it does if you haven't specified a PK explicitly. It also creates datetime fields for CreatedAt, UpdataedAt